src/macports1.0/macports_util.tcl

```diff
@@ -35,6 +35,36 @@ package provide macports_util 1.0
 # Provide some global utilities
 
 namespace eval macports_util {
+    ##################
+    # Public methods #
+    ##################
+
+    ##
+    # Given a list of maintainers as recorded in a Portfile, return an
+    # unobscured version that uses proper email addresses.
+    #
+    # @param list A list of obscured maintainers
+    # @return An unobscured list of maintainers
+    proc unobscure_maintainers {list} {
+        set result {}
+        foreach maintainer $list {
+            if {[string first "@" $maintainer] < 0} {
```

Comment:
But in this case that would result in the maintainer not getting into the result list. You could of course add it to the result before continuing.